pkgs/top-level/all-packages.nix

```diff
@@ -24075,7 +24075,7 @@ with pkgs;
 
   wavpack = callPackage ../development/libraries/wavpack { };
 
-  wayland = callPackage ../development/libraries/wayland { };
+  wayland = darwin.apple_sdk_11_0.callPackage ../development/libraries/wayland { };
```

Comment:
Is there a better solution on the way here? This is fine as a hotfix but I'd rather wayland wasn't using a special darwin callPackage long term…

Reply:
```suggestion
  wayland = callPackage ../development/libraries/wayland {
    stdenv = if stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
  };
```